notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -749,22 +749,15 @@
       "source": [
```

Comment:
<p>misplaced 'git' in description for sync</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>